src/pages/AlumniPage.jsx

```diff
@@ -1,90 +1,179 @@
-                <h1 className="text-black text-center font-inter text-4xl font-medium" style={{ letterSpacing: '8px' }}>
-                    HOW TO GET INVOLVED
-                </h1>
-                <p className="text-black text-center font-inter text-base leading-7 px-32">
-                    We strongly encourage all FUSION alumni to stay connected! There are many ways to stay involved and still make an impact on the FUSION space.
-                </p>
+            {/* alumni panel pic */}
+            <div className="w-full reletive z-10">
+                <img src={alum_panel}/>
             </div>
 
-            <div className="flex items-start max-w-[1270px] w-full h-[413px] mt-4">
-                <img src={PlaceholderPic} alt="Scholarship Placeholder" className="w-[626px] h-[413px] opacity-1"/>
-                <div className="ml-8 flex flex-col justify-center">
-                    <h2 className="text-black font-inter text-4xl font-medium pr-40" style={{ letterSpacing: '8px' }}>
-                        ALUMNI SCHOLARSHIP
-                    </h2>
-                    <p className="text-black font-inter text-base leading-7 pr-32 mt-4">
-                        Help us fund a yearly scholarship for a FUSION general member that has actively contributed to the community!
-                    </p>
+            {/* alumni scholarship */}
+            <div className="w-full h-[540px] bg-[#fff5eb] flex flex-row">
+                <div className="w-[60%] h-full relative">
+                    <div className="w-[62%] h-[76%] bg-white rounded-b-[60px] rounded-tl-[60px] ml-[27%] mt-[10%] z-10 relative border-2 border-white">
+                        <div className="w-[85%] ml-[7%]">
+                            <h1 className="font-[700] text-[#FFA523] mt-[9%] tracking-[.1em]">EMPOWERING FUTURES &gt;&gt;</h1>
+                            <h1 className="text-black text-3xl font-[700] mt-[3%] ">Alumni Scholarships for Tomorrow's Leaders</h1>
+                            <hr className="w-16 h-1  bg-[#FFA523] border-0 rounded mt-[5%]"/>
+                            <p className="text-xs text-black mt-[5%] leading-[22px]">FUSION is proud to offer alumni-sponsored scholarships, a testament to the strong support and dedication 
+                                of our alumni network. These scholarships are designed to empower current members by providing financial 
+                                assistance and recognizing their academic and professional achievements. Through the generosity of our 
+                                alumni, we continue to foster a culture of excellence and ambition, ensuring that every member has the 
+                                opportunity to succeed and make a lasting impact in the STEM fields.</p>
+                        </div>
+                    </div>
+                    <img src={alum_vector1} className="w-80 h-60 absolute transform translate-y-[-207%] z-0"/>
```

Comment:
It might be this one but I don't know if you notice but the top right vector isn't fully covering, you can see the slight line 

<parent container className="relative width-auto ...">
   <child image className="absolute top-... bottom-... left-... right-....">
</parent container>

^ an example of how you would do this